### PR TITLE
batch_ibv_post_send

### PR DESCRIPTION
**Pull request by lyppg**

### CODE REVIEW

File: src/libinfinistore.cpp [hunk 1]
```diff
@@ -719,39 +646,49 @@ int rw_rdma(connection_t *conn, char op, std::vector<block_t> &blocks, int block
...
-            ERROR("Failed to perform RDMA operation");
-            return -1;
+            // maybe we will use request ctx in the future
+            wr[i].wr_id = (uintptr_t)conn;
         }
+
+        wr[i].opcode = op == OP_RDMA_READ ? IBV_WR_RDMA_READ : IBV_WR_RDMA_WRITE;
+        wr[i].sg_list = &sge[i];
+        wr[i].num_sge = 1;
+        wr[i].send_flags = IBV_SEND_SIGNALED;
```
Comment (thesues) [inline src/libinfinistore.cpp:678]: it should be signaled every a few ops.  Not everyone

Comment (lyppg) [reply src/libinfinistore.cpp:678]: any document refer to this? 
each WR should generate a WC, right?

Comment (thesues) [reply src/libinfinistore.cpp:678]: from chatgpt. this is how we could reduce number of CQ events.

**Comment by thesues** [comment]
BAR1 related code is deprecated.

**Pull request closed without merging**